### PR TITLE
Feature#37 create incidents module

### PR DESCRIPTION
**Pull request by fego-dev**
# Description
What did you implemented/Why did you implemented this:
 - I coded a controller and model for incidents, this module is usually used for mobile users, the model is used for both them users mobile and web users.
 - Add method for get info about people tracking in 10 days past
 
 
  
  ## Screenshots 
imagen

### CODE REVIEW

**Review by LPMendoza — APPROVED** [review]
Good job bro, you code is clean and the commits are good

File: app/models/incident.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Incident < ApplicationRecord
+  belongs_to :user
+  has_one :user_history
+  has_one :test_covid
+  accepts_nested_attributes_for :test_covid, :user_history
+
+
+  def self.people_report(from = 10.days.ago.beginning_of_day.strftime('%F'), to = Date.today.strftime('%F'))
```
Comment (MiguelOcegueraM) [inline app/models/incident.rb:8]: Love it!

**Review by MiguelOcegueraM — APPROVED** [review]
Dude your code it's very clean! love it!

**Review by n0b0dy-su — APPROVED** [review]
Its a god implementation of RoR syntax an so clean

File: app/models/incident.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  private
+  def date_now
+    @now_date = Date.today.strftime('%F')
+  end
+
+  def date_from
+   @from_date = 10.days.ago.beginning_of_day.strftime('%F')
+  end
```
Comment (ArielMD) [inline app/models/incident.rb:29]: Are you using these methods? Above you have the same ones in "from" and "to"

**Review by ArielMD — APPROVED** [review]
nice job, everything looks good

**Pull request merged**